apps/api/R/inputs.R

```diff
@@ -0,0 +1,116 @@
+library(dplyr)
+
+#' Search for Inputs containing wildcards for filtering
+#' @param model_id Model Id (character)
+#' @param site_id Site Id (character)
+#' @param offset
+#' @param limit 
```

Comment:
The code below indicates that limit can only take on specific discrete values, but that's not in the function documentation

Reply:
It is there in the swagger documentation, the link to which is very clearly placed at the top of the Pecan Book page. These are just some examples

Reply:
Moreover, this API will be used tightly in conjunction with the `rpecanapi` package...in the corresponding function of `rpecanapi`, the documentation will clearly mention these specific values